docs/api-reference/self-signed-certificates.md

```diff
@@ -0,0 +1,79 @@
+<!--pytest-codeblocks:skipfile-->
+<!-- markdownlint-disable-next-line first-line-h1 -->
+When using the `quilt3` API or CLI with a client-to-site (remote
+access) VPN you may need to use a self-signed certificate to
+securely connect. This requires adding the certificate to your
+Python environment.
+
+## Mac OS X
+
+1. Open a new browser window and navigate to any HTTPS website.
+2. Click the «lock icon» in the browser navigation/address bar.
+3. Click «View certificates» and record the certificate name value
```

Comment:
```suggestion
3. Click View certificates and copy the certificate name to a safe place.
```

Reply:
"Click View certificates" without quotes kind of looks weird?

That said, we don't want to be too specific here since these are instructions for "any browser on OS X" - we're just hoping that any reasonable browser has such a feature.